cmd/containerd-stargz-grpc/main.go

```diff
@@ -75,6 +76,9 @@ type snapshotterConfig struct {
 
 	// NoPrometheus is a flag to disable the emission of the metrics
 	NoPrometheus bool `toml:"no_prometheus"`
+
+	// DebugAddress is a Unix domain socket address where the snapshotter exposes /debug/ endpoints.
+	DebugAddress string `toml:"debug_address"`
```

Comment:
Right now `debug` is already taken by a boolean flag. Does it make sense to have `debug` section as like containerd? It would be a breaking change though.

Reply:
We don't have many debug options as of now so having a flat field (debug_address) looks good to me. If we have too many debug options in the future, we can consider reorganizing the configuration fields (e.g. introducing debug section).